lib/generators/hyrax/templates/app/controllers/curation_concerns/admin_controller.rb

```diff
@@ -1,4 +1,4 @@
-module CurationConcerns
+module Hyrax
```

Comment:
Got back to this and fixed the path. Not going to bother nixing this file in this PR.